src/functions-and-arrays.js

```diff
@@ -29,9 +82,31 @@ const wordsUnique = [
   'bring'
 ];
 
+function uniquifyArray(arr){
+  if (arr.length == 0){
+    return null;
+  }
+  for(i=0;i<arr.length; i++){
+    if(indexOf(arr[i]).length>1){
+      arr.splice(i,1);
+    }
+  }
+ return arr;
+}
+
```

Comment:
I think the `.slice()` method is not a good approach to this case.
For this interaction, you can take a look at Jan solution:

`function uniquifyArray(ar) {
  if (ar.length === 0) return null;
  let uniqueArray = [];
  for (let el of ar) {
    if (uniqueArray.indexOf(el) === - 1) {
      uniqueArray.push(el)
    }
  }
  return uniqueArray;
}`